src/components/config-tabs/ConfigTabs.module.css

```diff
@@ -71,5 +71,12 @@ button {
 }
 
 .groupsContainer {
-    width: 100%;
+  width: 90%;
```

Comment:
Are you doing this to have less width between numerator and remove button?
I'd either  keep this at 100% or go smaller with a width defined in pixels and left alignment

Reply:
yeah, i was trying to avoid this big distance, but 100% is okay also given the fact that some groups names / numerator names might be long. 
![image](https://github.com/hisprwanda/who-data-quality-annual-report/assets/39731368/4a5c9ad7-b2aa-4e66-81f9-87afbc174528)